components/molecules/FilterHeader/filter-header.tsx

```diff
@@ -13,14 +14,18 @@ import FilterCardSelect from "components/molecules/FilterCardSelect/filter-card-
 import getTopicThumbnail from "lib/utils/getTopicThumbnail";
 import { interestsType } from "lib/utils/getInterestOptions";
 import { getInterestOptions } from "lib/utils/getInterestOptions";
+import Search from "components/atoms/Search/search";
 
 const HeaderFilter = () => {
   const router = useRouter();
   const filterOptions = useFilterOptions();
   const topicOptions = getInterestOptions();
 
   const { filterValues } = useFilterPrefetch();
-  const { filterName, toolName, selectedFilter } = router.query;
+  const { toolName, selectedFilter } = router.query;
+  const [filterName, setFilterName] = useState<string | string[] | undefined>(router.query.filterName);
+  console.log("filterName 1", filterName);
```

Comment:
nit: Please could you remove this line, or is it an intentional feature ???
```suggestion
```

Reply:
My bad. It was left over from debugging. I'll remove it in the morning.